src/licensedcode/data/licenses/cnpl-4.0.yml

```diff
@@ -0,0 +1,8 @@
+key: cnpl-4.0
```

Comment:
Can you rename this to a longer `cooperative-non-violent-4.0` ? the rationale is that less common license are entitled to more verbose license keys so they can stand out.